src/__tests__/list-test.tsx

```diff
@@ -37,10 +37,16 @@ test('Row as a button', () => {
 });
 
 test('Row with switch', () => {
```

Comment:
Looks like the test is not passing
![image](https://user-images.githubusercontent.com/579417/157918435-281932e7-40ce-4ff7-8745-b6a807441221.png)